Panic on startup with plugins.path containing subdirectories

**Describe the bug**

Running `inizio` with `--plugin.path`, which contains subdirectories, cause panic with index out of range.

The problem is in incorrect usage of `afero.Fs`: `RegexpFs` does not skip directories according to passed regexp:
```if i.IsDir() || f.re.MatchString(i.Name())```

**To Reproduce**

Steps to reproduce the behavior:
 - create directory `/usr/local/bin/inizio-plugins`
 - create directory `/usr/local/bin/inizio-plugins/test`
 - run `inizio`: `inizio /path/to/project`

**Expected behavior**

`inizio` must skip `FS` items according to regexp and not exit with panic.

**Environment:**

 - Version: [e.g. v1.1.0]
 - GOOS: [darwin]

**Additional context**

Panic stack trace:
```shell
panic: runtime error: index out of range [1] with length 0

goroutine 1 [running]:
github.com/insidieux/inizio/internal/plugins.(*Loader).Load(0xc0001d9bd0, 0x17c6df0, 0xc00028c6c0, 0x7ffeefbffb36, 0x2d, 0xc000322a20, 0x16f74ff, 0xc, 0x7ffeefbffb36, 0x2d)
	/project/internal/plugins/loader.go:51 +0x685
```

--- a/internal/plugins/loader.go
+++ b/internal/plugins/loader.go
@@ -46,10 +46,11 @@ func (l *Loader) Load(ctx context.Context, path string) ([]ClientInterface, erro
 
 	clients := make([]ClientInterface, 0)
 	for _, item := range list {
-		var (
-			parts = nameRegexp.FindStringSubmatch(item.Name())
-			name  = parts[1]
-		)
+		parts := nameRegexp.FindStringSubmatch(item.Name())
+		if len(parts) < 2 {
+			continue
+		}
+		name := parts[1]
 		cfg, err := l.config.Lookup(name)
 		if err != nil {
 			cfg = newPluginConfig(name)

--- a/internal/plugins/loader_test.go
+++ b/internal/plugins/loader_test.go
@@ -72,6 +72,7 @@ func (s *loaderTestSuite) TestLoad() {
 
 	filesystem := afero.NewMemMapFs()
 	_ = filesystem.MkdirAll(`/path`, 0755)
+	_ = filesystem.MkdirAll(`/path/dir`, 0755)
 	_, _ = filesystem.Create(`/path/inizio-plugin-first`)
 	_, _ = filesystem.Create(`/path/inizio-plugin-second`)
 

